src/metricscollector/db/metricscollector.db.changelog.yml

```diff
@@ -0,0 +1,33 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: byang
+      changes:
+        - createTable:
+            tableName: applicationmetrics
+            columns:
+              - column:
+                  name: appid
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(100)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: unit
+                  type: varchar(20)
+                  constraints:
+                    nullable: false                    
+              - column:
+                  name: timestamp
+                  type: bigint
```

Comment:
if you have multiple collectors, as long as the time is written in GMT, then yes, no timezone is needed.